.github/workflows/ci.yml

```
@@ -72,12 +72,11 @@ jobs:
 
     - name: Install fourmolu
       run: |
-        FOURMOLU_VERSION="0.14.0.0"
+        FOURMOLU_VERSION="0.18.0.0"
```

We can't use `0.17.0.0`, yet, because it cannot be build with ghc-9.6
```suggestion
        FOURMOLU_VERSION="0.17.0.0"
```

We also need to add fourmolu to `nix develop` and we will not be able to add `fourmolu-0.18.0.0` for that reason.

---

I think there is a typo in your message because you say we can't use 1.17 but suggest to change it to precisely 1.17.

In consensus we also support 9.6 and we do use 1.18. Perhaps our shell is based on 9.8?

Cc @amesgen

---

Yeah we [use GHC 9.8 to build fourmolu](https://github.com/IntersectMBO/ouroboros-consensus/blob/2c064714edc82636867ea8927faedbde5bec4aad/nix/tools.nix#L41) despite using GHC 9.6 as our main compiler version; no need to keep these in sync.

---

Feel free to ignore my comment then and also forgive my nix ignorance :see_no_evil: